qa-tests-backend/scripts/run-compatibility.sh

```diff
@@ -119,4 +124,34 @@ shorten_tag() {
     fi
 }
 
+verify_scannerV2_deployed() {
+    local namespace=${1:-stackrox}
+    info "Waiting for Scanner V2 deployment to appear in namespace ${namespace}..."
+    wait_for_object_to_appear "$namespace" deploy/scanner-db 600
+    wait_for_object_to_appear "$namespace" deploy/scanner 300
+    info "** Scanner V2 is deployed in namespace ${namespace}"
+}
+
+verify_scannerV4_deployed() {
+    local namespace=${1:-stackrox}
+    verify_scannerV4_indexer_deployed "$namespace"
+    verify_scannerV4_matcher_deployed "$namespace"
```

Comment:
How about keeping the same logging style?
```suggestion
    verify_scannerV4_indexer_deployed "$namespace"
    verify_scannerV4_matcher_deployed "$namespace"
    info "** Scanner V4 is deployed in namespace ${namespace}"
```

Reply:
Sure, but I think I'll be deleting the scannerV4 part after all.